stepup/start-dev-env.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Read the command line parameters to configure the dev env
+export SERVICE=$1
+export CODE_PATH=$2
+
+# Use docker compose to start the environment but with the modified override file
+docker-compose -f docker-compose.yml -f ./${SERVICE}/docker-compose.override.yml up
```

Comment:
Nice and clean, very understandable. Two things could be added:

1) Printing the actual command that is run. Very usefull for debugging, and also for the devs to see what's happening
2) A way to add all parameters that you'd add as parameters to the script. You can eg add -d to the docker-compose command, or other command line options:

So: start-dev-env.sh middleware ../middleware -d would do:
docker-compose -f docker-compose.yml -f ../middleware/docker-compose-override.yml up -d

Reply:
Thanks @quartje ! Nice ideas ;) I have implemented both them in my latest commit to this branch / PR.